templates/Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM node:12.16 AS JS_BUILD
 COPY webapp /webapp
-WORKDIR webapp
+WORKDIR /webapp
 RUN npm install && npm run build --prod
 
 FROM golang:1.13.8-alpine AS GO_BUILD
-RUN apk add build-base
+RUN  apk update && apk add --virtual build-dependencies build-base wget 
```

Comment:
Why do you add two additional packages if `build-base` is enough? Does it not work in some cases?

What would be the benefit of using `--virtual` here?

Reply:
build-base is not enough for my case.
```
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.11/main: temporary error (try again later)
WARNING: Ignoring APKINDEX.70f61090.tar.gz: No such file or directory
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.11/community: temporary error (try again later)
WARNING: Ignoring APKINDEX.ca2fea5b.tar.gz: No such file or directory
2 errors; 15 distinct packages available
ERROR: Service 'app' failed to build: The command '/bin/sh -c apk update && apk add  build-base' returned a non-zero code: 2
```
And I googled, found the solution here.
```
RUN  apk update && apk add --virtual build-dependencies build-base
```
I check again, the wget package is redundant, my mistake